browser_history/generic.py

```diff
@@ -95,9 +95,9 @@ def profiles(self) -> typing.List[str]:
         maybe_profile_dirs = []
         for profile_dir_prefix in self.profile_dir_prefixes:
             maybe_profile_dirs.extend(self.history_dir.glob(profile_dir_prefix))
-        profile_dirs = [profile_dir.name for profile_dir in maybe_profile_dirs
+        profile_dirs = [profile_dir.parent.name+'/'+profile_dir.name \
+                        for profile_dir in maybe_profile_dirs
```

Comment:
This would append the parent dir even if it's not needed and it wouldn't work if the directories are nested deeper. As mentioned in the issue #31 description, you need to implement a general solution that returns the relative path from `history_dir` regardless of any nesting.

Reply:
Correct me if Im wrong
I need to search for the correct path be it deep or single, and if it doesn't require any path addition let it be 
Right?
An example would be: now the current browsers need no path change so let it be while the test case provided needs a path addition , I need to build a generic code for it which adds path to the test case while leaving the current browser paths the same?

Reply:
Also Is it possible for you to close this pull request , I would like to open it during hacktober

Reply:
Yes, that's exactly right.

Reply:
You can close the PR yourself, there should be a button at the bottom